### PR TITLE
Feature #36 phases of pandemic screen

### LINKED ISSUE #36: Build out Phases of Pandemic and suggestions for self-care screen
At the bottom of the staff welfare screen, there is a button which links off to a "Phases of Pandemic and suggestions for self-care" screen

**We need to build build out the Build out Phases of Pandemic and suggestions for self-care screen**

@greggmiller Can you please provide some information in this issue on what should be in this section?  Once that is done assign the issue to Marc Edwards.

@marcedwards Once greg has provided info, can you provide some design guidance in this issue in on how it should look.  Once this is done, can you label this issue as "Ready for Dev" so that one of the devs can pick it up.

### LINKED ISSUE #215: Your Welfare - UI Inconsistencies
- [ ] A More spacing between heading and body
- [ ] B Emoji should align to top of each point

PPE 002

### PR DESCRIPTION
**Pull request by rockgecko-development**
Implement #36 phases of pandemic

## Screenshots

image

## Things to note

Reused existing ICUDailyRound classes and widgets, but added 2 subtitle fields for the "It's ok to not be OK", and "Edgy, poor sleep" in Long Term->PTSD tab.

### CODE REVIEW

**Comment by rockgecko-development** [comment]
Second commit changes the text size on this _and_ ICU daily round screen to match the mocks (from 24-17pt):
image

**Comment by rockgecko-development** [comment]
Third commit fixes the background colour:
image

File: lib/models/icu_daily_round_steps.dart [hunk 1]
```diff
@@ -21,7 +21,7 @@ class ICUDailyRoundStepSubsection {
 }
 
 class ICUDailyRoundItem {
-  final String icon, title;
+  final String icon, title, subtitle;
```
Comment (maks) [inline lib/models/icu_daily_round_steps.dart:24]: Would be good to make name casing consistent with subHeading

File: lib/view/staff_welfare/your_welfare_view.dart [hunk 2]
```diff
@@ -7,153 +7,45 @@ import '../../strings.dart';
...
-      ),
+          onPressed: () => UrlUtils.launchWithBrowser(whURL),
+        ),
+      ],
+      tabs: yourWelfareSteps.map((s) => s.heading).toList(),
+      children: yourWelfareSteps
+          .map((e) => ICUDailyRoundStepsContainer(
+                steps: e,
+                backgroundColor: AppColors.grey500,
+              ))
```
Comment (maks) [inline lib/view/staff_welfare/your_welfare_view.dart]: Please add trailing commas

**Review by maks — APPROVED** [review]
👍 great work @rockgecko-development ! Just a couple of small nits.

File: lib/hard_data.dart [hunk 3]
```diff
@@ -846,3 +846,170 @@ const List<ICUDailyRoundChecklist> icuDailyRoundChecklist = [
...
+        ICUDailyRoundItem(icon: '🛀', title: 'Sleep hygiene!'),
+      ])
+    ])
+  ]),
+  ICUDailyRoundSteps(heading: 'Core phase', sections: [
+    ICUDailyRoundStepSection(
+      heading: 'Core phase',
+      subsections: [
+        ICUDailyRoundStepSubsection(
+            heading: 'Number of case: Full scale - Multiple',
```
Comment (kimngyn) [inline lib/hard_data.dart]: "Full scale/Multiple" ? I think the design changed [and "cases" with an extra "s" at the end. I'm not sure if my suggestion went through]

File: lib/hard_data.dart [hunk 4]
```diff
@@ -846,3 +846,170 @@ const List<ICUDailyRoundChecklist> icuDailyRoundChecklist = [
...
+        ICUDailyRoundItem(
+            icon: '💤', title: 'Ensure the basics (as per previous page)'),
+      ])
+    ])
+  ]),
+  ICUDailyRoundSteps(heading: 'End phase', sections: [
+    ICUDailyRoundStepSection(
+      heading: 'End phase',
+      subsections: [
+        ICUDailyRoundStepSubsection(heading: 'Number of case: Few', list: [
```
Comment (kimngyn) [inline lib/hard_data.dart]:
```suggestion
        ICUDailyRoundStepSubsection(heading: 'Number of cases: Few', list: [
```

File: lib/hard_data.dart [hunk 5]
```diff
@@ -846,3 +846,170 @@ const List<ICUDailyRoundChecklist> icuDailyRoundChecklist = [
...
+        ICUDailyRoundItem(icon: '💡', title: 'Communication updates'),
+        ICUDailyRoundItem(icon: '🙋‍♀️', title: 'Employee wellbeing officer'),
+      ])
+    ])
+  ]),
+  ICUDailyRoundSteps(heading: 'Initial phase', sections: [
+    ICUDailyRoundStepSection(
+      heading: 'Initial phase',
+      subsections: [
+        ICUDailyRoundStepSubsection(heading: 'Number of case: 1', list: [
```
Comment (kimngyn) [inline lib/hard_data.dart]:
```suggestion
        ICUDailyRoundStepSubsection(heading: 'Number of cases: 1', list: [
```

File: lib/hard_data.dart (hunk 3 above)
Comment (kimngyn) [inline lib/hard_data.dart]:
```suggestion
            heading: 'Number of cases: Full scale - Multiple',
```

File: lib/hard_data.dart [hunk 6]
```diff
@@ -846,3 +846,170 @@ const List<ICUDailyRoundChecklist> icuDailyRoundChecklist = [
...
+            icon: '👏', title: 'Gratitude and thanks to all our colleagues'),
+      ])
+    ])
+  ]),
+  ICUDailyRoundSteps(heading: 'Long term', sections: [
+    ICUDailyRoundStepSection(
+      heading: 'Long term',
+      subsections: [
+        ICUDailyRoundStepSubsection(
+            heading: 'Number of case: 0 (or very low)',
```
Comment (kimngyn) [inline lib/hard_data.dart]:
```suggestion
            heading: 'Number of cases: 0 (or very low)',
```

File: lib/hard_data.dart [hunk 7]
```diff
@@ -792,7 +792,7 @@ List<ICUDailyRoundSteps> icuDailyRoundSteps = [
       ])
     ])
   ]),
-  const ICUDailyRoundSteps(heading: 'G - Git', sections: [
+  ICUDailyRoundSteps(heading: 'G - Git', sections: [
```
Comment (kimngyn) [inline lib/hard_data.dart:795]:
```suggestion
  ICUDailyRoundSteps(heading: 'G - GIT', sections: [
```

**Comment by rockgecko-development** [comment]
just working on finishing this up now

File: lib/hard_data.dart (hunk 3 above)
Comment (rockgecko-development) [reply lib/hard_data.dart]: Yep, designs for all these were changed after I implemented :+1:

**Pull request merged**